content/definition.md

```diff
@@ -1,27 +1,40 @@
 +++
-title = "The Ethical Source Definition (ESD)"
+title = "The Ethical Source Principles"
 template = "definition"
-heading="The Ethical Source Definition"
+heading="The Ethical Source Principles"
 +++
 
-Software is considered **Ethical Open Source** (EOS) when it meets *all* of these criteria:
+The practice of Ethical Source is informed by these principles:
 
-1. **It benefits the commons.** The license under which the software is released must not prohibit modification, derivative works, and linking or compiling with other software (unless under a copyleft provision), in the general spirit of open source, source available, or other commons distribution.
-2. **It is created in the open.** The source code must be publicly available, developed and maintained in public view, and welcoming of public contributions (subject to review and approval by the software project’s maintainers.)
-3. **Its community is welcoming and just.** The software project’s community of maintainers and contributors must publish clear rules for project governance and adopt a comprehensive code of conduct that is consistently and fairly enforced.
-4. **It puts accessibility first.** If the software has a user interface, it must be designed with accessibility in mind, ensuring all software functionality is available to all users, including users who may rely on assistive devices.
-5. **It prioritizes user safety.** The software must be designed with features and safeguards that minimize the risk of abuse or harm to others through the use of the software.
-6. **It protects user privacy.** If the operation of the software collects end-user data, the software must be designed with provisions for its operators to completely delete or provide to the end-user any and all data generated and stored from the end-user’s usage of the software. The export mechanism must provide the data in a human-readable or widely interoperable open format.
-7. **It encourages fair compensation.** The software project’s maintainers may, at their discretion, request remuneration in the form of code contributions, financial consideration, or other forms of voluntary support from organizations that derive commercial value from the software.
+1. **Our work benefits the commons.** We release our software under conditions that do not prohibit modification, derivative works, and similar use and reuse in the general spirit of free, open source, source available, or other commons distribution.
+2. **Our work is done in the open.** We make our source code publicly available, develop and maintain it in public view, and welcome  public contributions.
+3. **We strive to make our community welcoming and just.** We have clear and documented rules for governance, and we consistently and fairly enforce a comprehensive code of conduct.
```

Comment:
I'd be inclined to add something here about favoring autonomy of contributors, consensus decision-making, etc. But might be too specific for now.